hca2scea-backend/test/golden/arguments.csv

```diff
@@ -1,5 +1,2 @@
 spreadsheet,HCA project uuid,E-HCAD accession,curator initials,technology,experiment type,factor values,public release date,hca last update date,study accession
-Test01.xlsx,c893cb57-5c9f-4f26-9312-21b85be84313,50,AD,10Xv2_3,baseline,individual,2009-02-27,2021-04-29,SRP257542
-Test02.xlsx,2086eb05-10b9-432b-b7f0-169ccc49d270,52,AD,10Xv2_3,differential,organ part,2019-08-01,2021-03-01,SRP201293
-Test03.xlsx,d3a4ceac-4d66-4984-9704-2570c0647a56,54,AD,10Xv3_3,differential,organ part,2020-09-02,2021-03-01,SRP279608
-Test04.xlsx,559bb888-7829-41f2-ace5-2c05c7eb81e9,55,AD,10Xv2_3,differential,organ,2020-04-01,2021-03-01,SRP179761
+TEST01.xlsx,c893cb57-5c9f-4f26-9312-21b85be84313,50,AD,10Xv2_3,differential,disease,2009-02-27,2021-04-29,SRP151260

```

Comment:
Let's keep the files consistently named.
There were all named in CamelCase rather than ALL_CAPS, e.g. Test01 rather than TEST01.
This makes a mess in the scripts, so please go back to the original naming.